api/v1/helpers/verbs/findUtils.js

```diff
@@ -114,7 +126,11 @@ function toSequelizeWhere(filter, props) {
        */
       if (Array.isArray(props.fieldsWithEnum) &&
         props.fieldsWithEnum.indexOf(key) > -ONE) {
-        const enumArr = filter[key];
+
+        // if specified in props, convert the array in query to camelcase.
+        const enumArr = (props.fieldsToCamelCase &&
+          props.fieldsToCamelCase.indexOf(key) > -ONE) ?
+          convertArrayElementsToCamelCase(filter[key]) : filter[key];
 
```

Comment:
yes that the current status on master is the application throws an error with samples?status=A. Will create another story to give a more useful error message